app/timetables/models.py

```diff
@@ -1,3 +1,12 @@
 from __future__ import unicode_literals
 
 from django.db import models
+
+
+class Meal(models.Model):
```

Comment:
Same here, let's test this :)

https://realpython.com/blog/python/testing-in-django-part-1-best-practices-and-examples/